app/utils/graphql.ts

```diff
@@ -213,18 +217,3 @@ export const useLookupQuery = (lookupTableDef: LookupTableDef | undefined) =>
       typename,
     ];
   }, [lookupTableDef]);
-
-/**
- * Hook that persists queried data while new data is being fetched / revalidated.
- * this works as a complement to SWR by persisting data when fetch args
- * have changed
- **/
-export const useDataCache = <T>(currentData: T | null) => {
```

Comment:
turns out there was already an SWR option that does what `useDataCache` did:
https://swr.vercel.app/docs/advanced/understanding.en-US#return-previous-data-for-better-ux

👍